distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestPlatform.java

```diff
@@ -570,13 +581,113 @@ public void testRelativePathOutsideDdfHome() throws Exception {
 
         System.setProperty("javax.net.ssl.keyStore", "../cat.txt");
         String response = console.runCommand(EXPORT_COMMAND);
-        assertThat(String.format("Should not have been able to export to %s.",
-                getExportDirectory()),
-                response,
-                containsString(String.format("Failed to export all configurations to %s",
-                        getExportDirectory())));
+        // restore to initial state
         System.setProperty("javax.net.ssl.keyStore", "etc/keystores/serverKeystore.jks");
         testFile.delete();
+        assertThat(
+                String.format("Should not have been able to export to %s.", getExportDirectory()),
+                response, containsString(String.format("Failed to export all configurations to %s",
+                        getExportDirectory())));
+    }
+
+    /**
+     * Tests that when system properties file is missing, export fails
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testFailureWithoutSystemPropertiesFile() throws Exception {
+        FileUtils.deleteQuietly(getExportDirectory().toFile());
+
+        File systemProperties = Paths.get(ddfHome, "etc", "system.properties")
+                .toFile();
+
+        // back up sys.prop to restore after appending to it
+        File systemPropertiesCopy = Paths.get(ddfHome, "etc", "system.properties.copy")
+                .toFile();
+        FileUtils.copyFile(systemProperties, systemPropertiesCopy);
+        systemProperties.delete();
+
+        String response = console.runCommand(EXPORT_COMMAND);
+        // restore to initial state
+        FileUtils.copyFile(systemPropertiesCopy, systemProperties);
+        systemPropertiesCopy.delete();
+        assertThat(
+                String.format("Should not have been able to export to %s.", getExportDirectory()),
+                response, containsString("An error was encountered while executing this command."));
+
+    }
+
+    /**
+     * Tests that when system properties file is missing, export fails
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testFailureWithoutUsersPropertiesFile() throws Exception {
+        FileUtils.deleteQuietly(getExportDirectory().toFile());
+
+        File usersProperties = Paths.get(ddfHome, "etc", "users.properties")
+                .toFile();
+
+        // back up sys.prop to restore after appending to it
+        File usersPropertiesCopy = Paths.get(ddfHome, "etc", "users.properties.copy")
+                .toFile();
+        FileUtils.copyFile(usersProperties, usersPropertiesCopy);
+        usersProperties.delete();
+
+        String response = console.runCommand(EXPORT_COMMAND);
```

Comment:
Do you think the following would be cleaner/easier to understand? That would get rid of the comments and would also guarantee that if the cleanup happens even if the test logic (e.g., `runCommand()`) throws some kind of exception.

```
String response;
try {
    response = console.runCommand(EXPORT_COMMAND);
    assertThat(...);
}
finally {
    // Cleanup
}
```